index.html

```diff
@@ -225,6 +226,210 @@ <h1 class="playfair">Get Started</h1>
+                    </div>
+                </div>
+            </div>
+    
+            <div class="div2">
+                <h1 class="playfiar">About</h1>
+                <p class="poppins">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore 
+                    magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+                </p>
+                <button type="button" class="poppins">
+                    Get my CV
+                    <img src="img/ic_download.svg" alt="">
+                </button>
+            </div>
+        </section>
+    
+        <section class="desktop desktop-contact">
+            <div>
+                <div>
+                    <h1>Get Started</h1>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+                    magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+                </p>
+                <form action="https://formspree.io/f/xgedqaad" method="post">
+                    <ul class="ul-1">
+                            <li>
+                                <label for="nameD">Your name</label>
+                                <input 
+                                type="text" 
+                                name="user_name"
+                                class="inp-field" 
+                                id="nameD" 
+                                maxlength="30"
+                                required
+                                placeholder="Anastasia">
+                            </li>
+                            <li>
+                                <label for="emailD">Your email</label>
+                                <input 
+                                type="email"
+                                pattern="^([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$"
+                                name="user_email"
+                                class="inp-field" 
+                                id="emailD"                     
+                                maxlength="30"
+                                required>
+                            </li>
+                    </ul>
+                    <ul class="ul-2">
+                        
+                        <li>
+                            <label for="txt-msg">Message</label>
+                            <textarea name="user_message" id="txt-msg" class="text-msg" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                        </li>
+                    </ul>
+                    <button type="submit" class="button1">Start collaboration</button>    
+                </form>
```

Comment:
- [ ] **Required** - To be honest, you're rocking on this project 🚀 . Only minor color changes to maintain the design are required. Kindly ensure you apply the dark color (the one used on the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A553)) to the inputs, only when its in focus.

# Your current UI
![image](https://user-images.githubusercontent.com/35315311/178087788-e3b3b3cc-f28e-4023-b241-ed0e3940b8d2.png)

# vs Design

![image](https://user-images.githubusercontent.com/35315311/178087811-e0b4bfc3-71af-44ae-8386-4417b1bc5c9c.png)